requirements.txt

```diff
@@ -6,11 +6,11 @@ kafka-python~=2.0.2
 logzero>=1.7.0
 magic-logger>=1.0.2
 pandas>=1.2.4
-psycopg2-binary~=2.8.6
+psycopg2-binary~=2.9.3
 pyarrow==8.0.0
 python-json-logger~=2.0.1
 PyYAML~=5.4.1
 s3fs==0.4.2
 simplejson~=3.17.2
 SQLAlchemy>=1.4.11
-tables~=3.6.1
+tables~=3.7.0
```

Comment:
tables `3.6.1` was very outdated (released in Oct 2019) and it didn't support python 3.10. This has been resolved in the latest tables version which is `3.7.0`

There is an issue in their github explaining the above in more details: https://github.com/PyTables/PyTables/issues/909

Reply:
Same as `psycopg2-binary`; does it work for python 3.8?